ci/tests/token-transfer.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+HERMES_BIN=${HERMES_BIN:-${HOME}/build/ibc-rs/release/hermes}
+
+echo "Testing token transfer..."
+
+"${HERMES_BIN}" tx raw ft-transfer --dst-chain ibc-0 --src-chain basecoin-0 --src-port transfer --src-channel channel-0 \
```

Comment:
The latest API is `hermes tx ft-transfer` (`raw` removed); would it be better to just update it now so the tests don't break next time we update `hermes`?